src/main.rs

```diff
@@ -16,8 +16,18 @@ fn main() {
     if args.len() < 2 {
         panic!("problem id must be provided");
     }
-    let id = &args[1];
-    let id = id.parse::<u32>().expect(&format!("not a number: {}", id));
+    let id_arg = &args[1];
+    let mut id :u32 = 0;
+    match id_arg.as_ref() {
+        "random" => {
+            println!("You select random mode.");
+            id = get_random_id();
```

Comment:
What if the problem is locked or don't have a rust version?

Reply:
yes, current implementation has such limitation. 
Do you think we should verify whether the problem of generated id meet these two requirement at first? And regenerate an id that meets these two requirement

Reply:
In addition, using random mode to generate problems results in the disorder of lib.rs

Reply:
1. I think it is nice to have the ID re-generation, but a proper error message is also okay.
2. It's not a problem, we don't have an order guarantee for `lib.rs`